data/tests/tests_framework.txt

```diff
@@ -9,20 +9,23 @@
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 test "Test-Framework - Empty Testcase"
-	status "active"
+	# Partial; saving test-overhead by only running this test in combination with other test.
+	status partial
 	description "Test with no test sequence. Only starts and exits the game."
```

Comment:
I don't feel making these changes is necessary.

What we could do instead is move them into a plugin and include the plugin in the CI stage whenever framework code is modified.

Reply:
Yes, we could  move tests to their own plugins; I'll keep that for a follow-up PR. I restored the tests to their previous status in this PR.